docs/how_to/single_node.md

```diff
@@ -16,9 +16,11 @@ directory on your host for durability:
 
 ```
 docker pull quay.io/m3/m3dbnode:latest
-docker run -p 7201:7201 -p 9003:9003 --name m3db -v $(pwd)/m3db_data:/var/lib/m3db quay.io/m3/m3dbnode:latest
+docker run -p 7201:7201 -p 7203:7203 -p 9003:9003 --name m3db -v $(pwd)/m3db_data:/var/lib/m3db -v $GOPATH/src/github.com/m3db/m3/src/dbnode/config/m3dbnode-local-etcd.yml:/etc/m3dbnode/m3dbnode.yml quay.io/m3/m3dbnode:latest
```

Comment:
can we add a little comment about what each port being exposed does?

Reply:
I like the idea of just having a command you can copy and paste into your terminal - I'll make a note though if you don't have a GOPATH setup

Reply:
I really think you should remove the $GOPATH. Copy paste and working is fine but "copy and paste this config file into your local directory" is way easier than:

```
1. Install golang
2. Setup your $GOPATH
3. Clone the M3 repository and make sure you clone it into $GOPATH/src/....
```